Mathlib/LinearAlgebra/BilinearForm/Orthogonal.lean

```diff
@@ -166,7 +166,7 @@ theorem mem_orthogonal_iff {N : Submodule R M} {m : M} :
 theorem orthogonal_le (h : N ≤ L) : B.orthogonal L ≤ B.orthogonal N := fun _ hn l hl => hn l (h hl)
 #align bilin_form.orthogonal_le LinearMap.BilinForm.orthogonal_le
 
-theorem le_orthogonal_orthogonal (b : B.IsRefl) : N ≤ B.orthogonal (B.orthogonal N) :=
+theorem le_orthogonal_orthogonal (b : B.IsRefl) : N₂ ≤ B.orthogonal (B.orthogonal N₂) :=
```

Comment:
Why the change to N2 here?

Reply:
Good question! I think it's a hangover from before we made `R` commutative https://github.com/leanprover-community/mathlib4/commit/347b765edc635a9b8fa132661b2dc3e2c3e29fc1 I've removed it in favour of `N` now.